### PR TITLE
updated private key backup caution message

### PR DESCRIPTION
**Pull request by maeliosakyne**
## Summary
Updated the private key backup caution box, so that the text is more clear on what exactly should be backed up 

ticket:
https://github.com/ObolNetwork/obol-docs/issues/283

### CODE REVIEW

File: docs/int/quickstart/group/quickstart-group-leader-creator.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ You should expect to see a console output like
 If instead of being shown your `enr` you see an error saying `permission denied` then you may need to [update docker permissions](/docs/int/faq/errors#docker-permission-denied-error) to allow the command to run successfully.
 
 :::caution
-Please make sure to create a backup of the private key at `.charon/charon-enr-private-key`. Be careful not to commit it to git! **If you lose this file you won't be able to take part in the DKG ceremony and start the DV cluster successfully.**
+Ensure you create a backup of the private key stored in the '.charon' folder, specifically at '.charon/charon-enr-private-key'. This is the file used to generate your private key. Be careful not to commit it to git! **If you lose this file you won't be able to take part in the DKG ceremony and start the DV cluster successfully.**
```
Comment (gsora) [inline docs/int/quickstart/group/quickstart-group-leader-creator.md:66]: > This is the file used to generate your private key

We should clarify that this file doesn't generate the *DV* private key, but contains the *cluster identity key*.

Comment (thomasheremans) [reply docs/int/quickstart/group/quickstart-group-leader-creator.md:66]: We should also add that they should save the private key as it is used to generate the ENR. Just saving the ENR is useless which is what some operators have been doing as it displays a warning in the CLI to save it. @gsora we should maybe also clarify the CLI?

Comment (gsora) [reply docs/int/quickstart/group/quickstart-group-leader-creator.md:66]: > We should also add that they should save the private key as it is used to generate the ENR.

💯 

> we should maybe also clarify the CLI?

When you create an ENR with `charon create enr` we already print a pretty sizable caution message.

Do you think we should change something with it?

Screenshot 2023-11-27 at 11 23 31

Comment (thomasheremans) [reply docs/int/quickstart/group/quickstart-group-leader-creator.md:66]: Yes!

**Pull request merged**